static/css/inline-selection.scss

```diff
@@ -191,70 +191,84 @@ $very-light-grey: #ededed;
 }
 
 .download-only {
-  position:absolute;
-  z-index: $overlay-z-index + 1;
-  bottom: 10px;
-
-  /* horizontal centering */
   left: 50%;
+  position: fixed;
   transform: translate(-50%, 0);
-
-  padding: 10px 15px;
-  border-radius: 100px;
-  background: #737373;
-  font-family: -apple-system, BlinkMacSystemFont, "segoe ui", "helvetica neue", helvetica, ubuntu, roboto, noto, arial, sans-serif;
-  font-weight: bold;
-  color: #fff;
-  font-size: 14px;
-  line-height: 26px;
-  white-space: nowrap;
+  transition: top 125ms ease-out, translate 125ms ease-out;
   user-select: none;
-}
+  will-change: top, translate;
```

Comment:
not sure if this makes a difference here, but telling the browser that we expect these to move may add a bit of smoothness.